website/source/docs/platform/k8s/injector/index.html.md

```diff
@@ -0,0 +1,264 @@
+  `vault.hashicorp.com/agent-inject-template-foobar`. This should map to the same
+  unique value provided in `vault.hashicorp.com/agent-inject-secret-`. If not provided,
+  a default generic template is used.
+
+* `vault.hashicorp.com/role` - configures the Vault role used by the Vault Agent
+  auto-auth method.  Required when `vault.hashicorp.com/agent-configmap` is not set.
+
+* `vault.hashicorp.com/agent-configmap` - name of the configuration map where Vault
+  Agent configuration file and templates can be found.
+
+* `vault.hashicorp.com/agent-pre-populate` - configures whether an init container
+  is included to pre-populate the shared memory volume with secrets prior to the
+  containers starting.
+
+* `vault.hashicorp.com/agent-pre-populate-only` - configures whether an init container
+  is the only injected container. If true, no sidecar container will be injected
+  at runtime of the pod.
+
+* `vault.hashicorp.com/agent-image` - name of the Vault docker image to use. This
+  value overrides the default image configured in the controller and is usually
+  not needed.  Defaults to `vault:1.3.1`.
+
+* `vault.hashicorp.com/service` - name of the Vault service to use.  This value
+  overrides the default service configured in the controller and is usually not
+  needed.
+
+* `vault.hashicorp.com/agent-limits-cpu` - configures the CPU limits on the Vault
+  Agent containers.  Defaults to `500m`.
+
+* `vault.hashicorp.com/agent-limits-mem` - configures the memory limits on the Vault
+  Agent containers.  Defaults to `128Mi`.
+
+* `vault.hashicorp.com/agent-requests-cpu` - configures the CPU requests on the
+  Vault Agent containers.  Defaults to `250m`.
+
+* `vault.hashicorp.com/agent-requests-mem` - configures the memory requests on the
+  Vault Agent containers.  Defaults to `64Mi`.
+
+* `vault.hashicorp.com/tls-secret` - name of the Kubernetes secret containing TLS
+  Client and CA certificates and keys.  This is mounted to `/vault/tls`.
+
+* `vault.hashicorp.com/tls-server-name` - name of the Vault server to verify the
+  authenticity of the server when communicating with Vault over TLS.
+
+* `vault.hashicorp.com/tls-skip-verify` - configures the Vault Agent to verify
+  Vault's TLS certificate.
+
+* `vault.hashicorp.com/ca-cert` - path of the CA certificate used to verify Vault's
+  TLS.
+
+* `vault.hashicorp.com/ca-key` - path of the CA public key used to verify Vault's
+  TLS.
+
+* `vault.hashicorp.com/client-cert` - path of the client certificate used when
+  communicating with Vault via mTLS.
+
+* `vault.hashicorp.com/client-key` - path of the client public key used when communicating
+  with Vault via mTLS.
+
+* `vault.hashicorp.com/client-max-retries` - configures number of Vault Agent retry
+  attempts when 5xx errors are encountered.  Defaults to 2.
+
+* `vault.hashicorp.com/client-timeout` - configures the request timeout threshold,
+  in seconds, of the Vault Agent when communicating with Vault.  Defaults to `60s`.
```

Comment:
This is slightly ambiguous to me. Does it match Vault's pattern, e.g. these are all the same and accepted: 60, 60s, 1m ?

Reply:
Yeah, I'll clarify this!

Reply:
Fixed!